Tasks/UsePythonVersionV0/task.json

```diff
@@ -1,123 +1,124 @@
 {
-    "id": "33C63B11-352B-45A2-BA1B-54CB568A29CA",
-    "name": "UsePythonVersion",
-    "friendlyName": "Use Python version",
-    "description": "Use the specified version of Python from the tool cache, optionally adding it to the PATH",
-    "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/tool/use-python-version",
-    "helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?linkid=871498) or [see the Python documentation](https://www.python.org/doc/)",
-    "category": "Tool",
-    "runsOn": [
-        "Agent",
-        "DeploymentGroup"
+  "id": "33C63B11-352B-45A2-BA1B-54CB568A29CA",
```

Comment:
Are all of these changes relevant?
Could you please remove redundant indent changes?